src/include/daos/tests_lib.h

```diff
@@ -227,7 +222,7 @@ dts_sgl_init_with_strings(d_sg_list_t *sgl, uint32_t count, char *d, ...);
  */
 void
 dts_sgl_init_with_strings_repeat(d_sg_list_t *sgl, uint32_t repeat,
-	uint32_t count, char *d, ...);
+				 uint32_t count, char *d, ...);
```

Comment:
Alignment

Reply:
This one is github though, I've just checked it.